apiary.apib

```diff
@@ -3400,7 +3400,7 @@ Get a list of tasks.
     + Attributes (object)
         + filter (object, optional)
             + ids: `9c475de2-1e99-0328-bc19-271b27e921d8`,`6325b4e5-c6a6-060b-b010-b81892e82fa7` (array[string], optional)
-            + user_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional) - Returns Tasks that are assigned to this user or to a team to which this user belongs.
+            + user_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional, nullable) - Returns Tasks that are assigned to this user or to a team to which this user belongs. When passing `null`, returns tasks that are unassigned.
```

Comment:
PP on the English comment: I would rephrase this as `When passing null, it returns tasks` instead of `When passing null, returns tasks`. Also, the beginning of the sentence is `Returns Tasks that are assigned to this user`; I don't see why we use a capital letter for tasks (`Returns Tasks`), I would use `Returns tasks` (I realize this isn't something introduced in this PR, but it grabbed my attention).

Reply:
```suggestion
            + user_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional, nullable) - Returns tasks that are assigned to this user or to a team to which this user belongs. When passing `null`, it returns tasks that are unassigned.
```